### PR TITLE
Main-front

### LINKED ISSUE #1: Crear front de junior landing
### Explanation
Crear el front de junior landing primera parte dando las opciones en la parte superior que su función sera llevar al la parte opcional 

### Screeshoots
image

### PR DESCRIPTION
**Pull request by joselyepez**
Main
the logo is temporary

### CODE REVIEW

File: components/Main.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+          <p className={styles.caption}>
+            Te ofrecemos un ambiente donde puedas practicar trabajar en tech,
+            para que ganes experiencia simulando un ambiente de trajo real.
+          </p>
+          <br></br>
+          <div className={styles.containerIcon}>
+            <div className={styles.icon}>
+              <h2>200K+</h2>
+              <br></br>
```
Comment (Israel-Laguan) [inline components/Main.jsx]:
```suggestion
              <br/>
```

Comment (Israel-Laguan) [reply components/Main.jsx]: Same for the other cases

File: components/Main.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import MainLogo from "../public/MainLogo.png";
+import styles from "../styles/Main.module.css";
+
+const Main = () => {
+  return (
+    <div className={styles.container}>
+      <div className={styles.containerMain}>
+        <div className={styles.title}>
+          <h1>
+            Cada <span className="text-[#5651e5]"> Junior </span>es
```
Comment (Israel-Laguan) [inline components/Main.jsx]: Move those colors like `#5651e5` to a variable or select one closer in https://tailwindcolor.com/

File: components/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import Link from "next/link";
+import Image from "next/image";
+import styles from "../styles/Navbar.module.css";
+import logo from "../public/logo.png";
+
+const Navbar = () => {
+  return (
```
Comment (Israel-Laguan) [inline components/Navbar.jsx]:
```suggestion
const Navbar = () => (
```

File: pages/_app.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import Navbar from '../components/Navbar';
+import Main from '../components/Main'
+
+import '../styles/globals.css';
+
+function MyApp({ }) {
+  return (
```
Comment (Israel-Laguan) [inline pages/_app.js]:
```suggestion
const MyApp = () => (
```

File: styles/Navbar.module.css [hunk 5]
```diff
@@ -0,0 +1,11 @@
+  .item {
+    @apply ml-10 text-sm uppercase hover:border-b font-bold;
+  }
+
+  .container{
```
Comment (Israel-Laguan) [inline styles/Navbar.module.css]:
```suggestion
  .container {
```

File: styles/Navbar.module.css [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+ @apply fixed w-full h-20 shadow-xl z-[100] bg-[#2BA84A] ;
```
Comment (Israel-Laguan) [inline styles/Navbar.module.css]:
```suggestion
     @apply fixed w-full h-20 shadow-xl z-[100] bg-[#2BA84A] ;
```

File: components/Main.jsx (hunk 2 above)
Comment (Israel-Laguan) [reply components/Main.jsx]: Move these colors to custom colors

https://tailwindcss.com/docs/customizing-colors#using-custom-colors

**Comment by Israel-Laguan** [comment]
Its giving this error, please check 
image

Seems like you need rebase
image

**Comment by joselyepez** [comment]
> Está dando este error, verifique imagen
> 
> Parece que necesitas rebase imagen

I was getting an error when doing commit and I did it several times
That error is from Cloudflare because they didn't configure it previously

**Comment by Israel-Laguan** [comment]
You need to rebase

If you need help, send me a message so we can meet

**Review by elias-miguel-vergara — COMMENTED** [review]
revisado esperar los cambios para aprobar

**Review by elias-miguel-vergara — APPROVED** [review]

**Review by Jesus-Gonzalez-Arroyo — APPROVED** [review]

File: components/Main.jsx (hunk 2 above)
Comment (Israel-Laguan) [reply components/Main.jsx]: @joselyepez don't forget to change this

File: components/Navbar.jsx [hunk 7]
```diff
@@ -5,6 +5,30 @@ import styles from "../styles/Navbar.module.css";
...
+          </Link>
+          <Link href="/">
+            <li className={styles.item}>Sobre Nosotros</li>
+          </Link>
+        </ul>
+      </div>
+    </div>
+  </div>
+);
+=======
```
Comment (Israel-Laguan) [inline components/Navbar.jsx]: Check this, seems like you forgot to adapt the merge message

File: styles/Main.module.css [hunk 8]
```diff
@@ -23,9 +23,9 @@
 }
 
 .colorB {
-    @apply text-[#1D4ED8];
+    @apply text-[#1d4ed8];
```
Comment (Israel-Laguan) [inline styles/Main.module.css]: I think this name for the color is not good, maybe think of a better name like "dark-gray", "light-purple" or "spark-yellow"

File: components/Navbar.jsx (hunk 7 above)
Comment (Israel-Laguan) [reply components/Navbar.jsx]: This is causing the error in the build

File: styles/Main.module.css (hunk 8 above)
Comment (joselyepez) [reply styles/Main.module.css]: check new color @Israel-Laguan

File: styles/Main.module.css [hunk 9]
```diff
@@ -23,7 +23,7 @@
 }
 
 .colorB {
-    @apply text-[#1d4ed8];
```
Comment (Israel-Laguan) [inline styles/Main.module.css]: please change the class name, or move the color to a variable as described here https://tailwindcss.com/docs/customizing-colors#using-custom-colors

File: styles/Main.module.css (hunk 8 above)
Comment (Israel-Laguan) [reply styles/Main.module.css]: La vaina no es cambiar en color, sino cambiar el nombre de la clase que dice colorB y es como raro ese nombre

Comment (joselyepez) [reply styles/Main.module.css]: I understood to change the color palette

Comment (Israel-Laguan) [reply styles/Main.module.css]: Sorry for the misunderstanding

**Comment by joselyepez** [comment]
@Israel-Laguan please review

**Review by Israel-Laguan — APPROVED** [review]

**Pull request merged**